canonical-web-and-design/ubuntu.com
# Android on ubuntu takeover/engage page - due 14 August 2019

## Done

- Developing android on ubuntu takeover, engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- see that:
    - it is on the /engage page - http://0.0.0.0:8001/engage
    - the takeover looks correct and goes to the right engage page here is the [copy doc](https://docs.google.com/document/d/10w-f6FEEYGPGcrFFPiZmWioZnfC3lfvfjqx_tMsi1nE/edit) - http://0.0.0.0:8001/engage/developing-android-on-ubuntu here is the [brief](https://docs.google.com/spreadsheets/d/16tDtjCc9V29__YF5-5NTOkdX2Xp5sMXniaMRtNP-h_g/edit#gid=2113339190)
    - that the engage page copy, meta data and form are correct (*note the download isn't on the thank you form*)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1496

## Screenshots

### takeover
![image](https://user-images.githubusercontent.com/441217/62776889-bb84f800-baa3-11e9-8b1e-f17591fcf7e6.png)


### engage page
![image](https://user-images.githubusercontent.com/441217/62776852-a1e3b080-baa3-11e9-8b46-4c07d87ec6de.png)

### engage overview
![image](https://user-images.githubusercontent.com/441217/62776808-88daff80-baa3-11e9-9c02-4d4ce4c7214b.png)
